src/sentry/utils/snowflake.py

```diff
@@ -1,9 +1,16 @@
+import secrets
 from dataclasses import dataclass
-from datetime import datetime
-from typing import OrderedDict
+from datetime import datetime, timedelta
+from typing import OrderedDict, Tuple
 
 from django.conf import settings
 
+from sentry.utils import redis
+
+BASE_SEED = secrets.randbits(64)
```

Comment:
This can be deleted now that we've jettisoned the "assignment order shuffling" idea.